metrics/scaling/k8s_scale.sh

```diff
@@ -156,6 +171,26 @@ EOF
 	fi
 	metrics_json_end_nested_array "launched_pods"
 
+	# And store off the total memory consumed across all nodes, and the pod/Gb value
+	if [ $n_pods -eq 0 ]; then
+		local pods_per_gb=0
+	else
+		local pods_per_gb=$(bc -l <<< "scale=2; ($total_mem_used/1024) / $n_pods")
+	fi
+	local mem_json="$(cat << EOF
+			"memory": {
+				"consumed": {
+					"Result": ${total_mem_used},
+					"Units": "Kb"
+				},
+				"pods_per_gb": {
+					"Result": ${pods_per_gb}
```

Comment:
The value for "Result": was not being populated when I tried it. I set them all to 0 so I could see the results.
The key and colon are added, but not the ${pods_per_gb}. I believe it was set for the first iteration, but none after.

So the don't divide by 0 case worked, but the else was not populating the value.

Just checked, bc is not installed. I'll have to add that to my config patch.

Reply:
Looking into the gb/pod issue.
I've added a call to `check_cmds` (from our lib file) during initialisation to pick up a missing `bc` and report it.
I think it was the missing `bc` that caused the empty results fields, so hopefully that is fixed now..

Reply:
yes it was bc, I should have added that as multiple comments rather than editing the same one. My voyage of discovery wasn't clear.